opcua/server/server.py

```diff
@@ -318,12 +318,12 @@ def get_namespace_index(self, uri):
         uries = self.get_namespace_array()
         return uries.index(uri)
 
-    def get_event_object(self, etype=ua.ObjectIds.BaseEventType, source=ua.ObjectIds.Server):
+    def get_event_object(self, event=ua.BaseEvent()):
```

Comment:
And also I don't think that name `get_event_generator` is better than `get_event_object`. Probably the most correct would be `generate_event` but anyway I will rename it to `get_event_generator`